### DISCUSSION TITLE
Rename JUnit fixtures in report

### DISCUSSION THREAD

**Discussion by ghost** [allure-framework/allure2#1995] [Questions & Support] [answered]
Is it possible to rename JUnit fixtures method names in the report?

For example

  `@AfterEach
      public void tearDown() {}`

displays as tearDown and @DisplayName isn't working here.

Allure JUnit 5 2.22.0

**Comment by baev** [comment, marked as answer]
the issue https://github.com/allure-framework/allure-java/issues/554
